brozzler/cli.py

```diff
@@ -131,30 +142,58 @@ def decorate_logger_name(a, b, event_dict):
     return event_dict
 
 
+# https://manpages.debian.org/testing/libsystemd-dev/sd-daemon.3.en.html
+def systemd_log_prefix(_, level, log):
```

Comment:
consider indicating private visibility: 

```suggestion
def _systemd_log_prefix(_, level, log):
```

Admittedly nittish considering it's very unlikely anyone's going to reuse this function outside the context of logging, but I always like to make it perfectly clear that one would be doing something wrong using a private symbol outside of the enclosing module scope.

Reply:
Good call!